frontend/src/Judge/components/NavBar.js

```diff
@@ -35,6 +37,9 @@ export default class NavBar extends Component {
           <NavbarToggler onClick={this.toggle} />
           <Collapse isOpen={this.state.isOpen} navbar>
             <Nav className="ml-auto" navbar>
+              <NavItem>
+                <NavLink tag={Link} to='/' onClick={this.props.switchToAdmin}>Switch to: Admin</NavLink>
```

Comment:
I would rename this to "View as Admin"

Reply:
Also, I would add some sort of visual separation between this and the other links, say a `|` symbol in between them.